Automattic/wp-calypso
# search: Convert to function component

#### Changes proposed in this Pull Request

* Converts the `Search` component to be a function component.

#### Testing instructions



* Run `search:storybook:start` and test the search component.
* Open `/plugins` and ensure that search still works
* Open `/people/edit/:site/:user` where `site` is a site with more than 10 users. Scroll to the bottom and select the option to attribute works to another user and ensure search still functions.



Related to #
